src/ngx_lua_resty_lmdb_module.c

```diff
@@ -1,13 +1,30 @@
+#include <openssl/evp.h>
+
 #include <ngx_lua_resty_lmdb_module.h>
 
 
+#define ENC_KEY_LEN         32
+
+#ifndef EVP_DIGEST_CONSTANT
+#define EVP_DIGEST_CONSTANT "konglmdb"
```

Comment:
Shall we use a random string?

Reply:
we can type a random string here yes. but since this repo is open sourced, meaning no matter how "random" this is, someone could still generate a rainbow table around it; it's the same effort someone start to generate rainbow table with salt "konglmdb" 😄 

i was thinking if we should specify the macro at kong-build-tools at build time, and change between versions, but that might be too far and migrations needs some thinking. and also use a sane KDF instead of salted MD5. maybe for now let's just keep it as is.

Reply:
great